### ISSUE TITLE
[Security] Harden pgaudit For Gids Database in Prod

### ISSUE THREAD

**Issue by Efe-Oddball** [department-of-veterans-affairs/va.gov-team#95495]
## Description
DB scans from July reported that pgaudit.conf needs to be reviewed and updated to ensure we capture everything needed.
These updates have been implemented in the Dev and Staging environments for the Gids database.

## Acceptance Criteria
- [x] _pgaudit.conf configured and hardened for Gids database in  prod_
- [ ] _Ensure DB scans captures the necessary audit data_

## Refinement Guidance - Check the following before working on this issue: 
- [ ] _Epic assigned (if needed)_ 
- [ ] _Estimated (points assigned)_
- [ ] _Sprint assigned (once planned)_
- [ ] _Team member(s) assigned_
- [ ] _Understands how this work aligns with the overall platform_

**Comment by Efe-Oddball** [comment]
Waiting to schedule security re-scan to ensure that audit details are being logged

**Comment by Efe-Oddball** [comment]
Due to the effects of the audit logs to the functionality of the database, we have to go back to the drawing board to discuss what needs to be done with adding audit logs to the prod databases. The audit logs tend to fill up the database memory really really fast. This ticket will be on hold, until we discuss the way forward with the team.

**Comment by Efe-Oddball** [comment]
Due to the effect of audit logs on database storage, I am moving this ticket to a blocked state until the database discovery ticket attached to this issue is resolved

**Comment by Efe-Oddball** [comment]
I have been working on preparation to implement audit logs in Gids prod database and remove logging of the application user READS to help with the effect of audit logs on database instance storage

**Comment by Efe-Oddball** [comment]
I have created Datadog monitor for the Gids database storage that we will have to watch for a few days to a week after audit log implementation, to ensure that there are no spikes on the storage being used

**Comment by Efe-Oddball** [comment]
Pushed out PR to implement audit logs

**Comment by Efe-Oddball** [comment]
Here is the [documentation](https://docs.google.com/document/d/1DaARBCgQfWgw_S6sr5--c1_LjAmEYkwxbruz1wBEPOA/edit?tab=t.0) for removing application user READS from RDS Instance

**Comment by Efe-Oddball** [comment]
<img width="908" alt="Image" src="https://github.com/user-attachments/assets/915daaef-432e-4100-a37b-c2b68ced83cc" />

**Comment by Efe-Oddball** [comment]
This confirms that pgaudit is active in the database

<img width="638" alt="Image" src="https://github.com/user-attachments/assets/6499fd8c-2f52-4837-bc06-a0a1fcbb2235" />

**Comment by Efe-Oddball** [comment]
I will observe the effect of the audit on storage in the coming days and request a security review to confirm that this database already passes the pg_audit scan

**Comment by Efe-Oddball** [comment]
waiting on result of scan before resolving this ticket

**Comment by Efe-Oddball** [comment]
The scan team recently migrated their scanner to on-prem with a new IP range and they are currently not able to access the PORTS to implement a scan. We have given them full access from our end but the ports are still closed. This is a result of VAEC transit Gateway or TIC ports. We are trying to reach out to the appropriate team to get this resolved, so the scan team can conclude the scan and this ticket can be resolved

**Comment by Efe-Oddball** [comment]
Due to the issue with the DB scan, I am moving this ticket to a resolved state. We will have to submit an ESECC request to get the scanners working with the database again. Moving this ticket to a resolved state
